ModelLibrary/Process/ViennaAdvantageProcess/LeadToOpportunity.cs

```diff
@@ -63,7 +63,11 @@ protected override String DoIt()
                 opp.SetC_EnquiryRdate(lead.GetC_EnquiryRdate());
                 if (lead.GetC_ProposalDdate() != null)
                 {
-                    opp.SetC_ProposalDdate(Convert.ToDateTime(lead.GetC_ProposalDdate()).ToLocalTime());
+                    /** Fix for Proposal Due Date issue ** Dt: 05/12/2021 ** Modified By: Kumar **/
+                    if (Env.IsModuleInstalled("WMN01"))
```

Comment:
Please plly VA077_ module check instead of WMN01
And Use Util.GetDateTime instead of convert.toDateTime